ivadomed/prepare_dataset.py

```diff
@@ -0,0 +1,215 @@
+import nibabel as nib
```

Comment:
Would it be possible to rename the file `preprocessing.py` to match `postprocessing.py`?
If too specific to a project, we could also move some functions to the dev folder.

Reply:
I think it would be more fitting to put it in the dev folder it's not really 'preprocessing' it's mode the creation of the dataset :)

Reply:
I was thinking eg some image normalisation tools or multivariate gaussians that could find a spot in `transforms.py` or `postprocessing.py`. Only a suggestion, your call :-)

Reply:
yes perhaps! Multi variate gaussian mostly might be useful for detection and not hidden here. I'll think about it.